sdk/python/tests/test_QrStorage.py

```diff
@@ -49,7 +47,7 @@ def test_cannot_save_more_than_max_buffer(self):
 			buffer = TestUtils.randbytes(MAX_DATA_BYTES + 1)
 
 			# Act + Assert:
-			with self.assertRaises(qrcode.exceptions.DataOverflowError):
+			with self.assertRaises(ValueError):
```

Comment:
why this change - did exception change?

Reply:
Yea, in v7.4 - https://github.com/lincolnloop/python-qrcode/blob/b80fea6ee7e75f3024b9ed7adf891a143e0b14e3/qrcode/util.py#L183